tagstudio/src/qt/ts_qt.py

```diff
@@ -508,6 +513,15 @@ def start(self):
         self.splash.finish(self.main_window)
         self.preview_panel.update_widgets()
 
+        # Autosave
+        self.autosave_timer = QTimer(self)
+        self.autosave_timer.timeout.connect(lambda: self.backup_library())
+        self.autosave_timer.setSingleShot(False)
+        self.updateSettings()
+
+        # Give settings to classes that need it.
+        ThumbRenderer.settings = self.settings
```

Comment:
it would be much cleaner to pass the existing `QSettings` instance  as a parameter to constructor, or pass the whole `QTDriver` which has the `.settings` property similarly as `PreviewPanel` does it:

https://github.com/TagStudioDev/TagStudio/blob/b8d72a65c8198d0840af3afd6ad33c1d67e378b0/tagstudio/src/qt/ts_qt.py#L443

so it would be something like:
```
# self refers to `QtDriver`
self.renderer = ThumbRenderer(self)

# then in ThumbRenderer:
def __init__(self, driver):
    self.driver = driver

# and then...
if self.driver.settings.contains("render_thumbnails") and self.driver.settings.value("render_thumbnails") == "true":

 


```